packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js

```diff
@@ -15,31 +17,39 @@ const theme = {
     },
     extend: props => {
       const { theme: { global: { colors } }, completeness } = props
+      const color = normalizeColor(colors['neutral-4'], props.theme)
+      const secondaryColor = getGradientShade(color)
       const percentComplete = `${completeness}%`
       const progressGradient = [
-        `${colors['neutral-4']}`,
-        `${colors['status-critical']} ${percentComplete}`,
-        `white ${percentComplete} calc(${percentComplete} + 1px)`,
-        `${colors['neutral-4']} ${percentComplete}`
-      ]
-      const hoverGradient = [
-        `${colors['neutral-4']}`,
-        `${adjustHue(7, colors['status-critical'])} ${percentComplete}`,
-        `white ${percentComplete} calc(${percentComplete} + 1px)`,
-        `${adjustHue(-7, colors['neutral-4'])} ${percentComplete}`
-      ]
+        `${colors['accent-4']} ${percentComplete}`,
+        `transparent ${percentComplete}`
+      ].join(',')
 
       return `
-        background: linear-gradient(to right, ${progressGradient.join(',')});
+        background: ${color};
+        box-shadow: 0px 2px 6px rgba(0, 0, 0, .3);
+        position: relative;
```

Comment:
Is the parent of this positioned? I'm trying to figure out why this is relatively positioned and what it's positioned relative to.

Reply:
Its `:before` is absolutely positioned relative to it